app/controllers/api/v1/revenue_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::RevenueController < ApplicationController
+  def index
+    revenue = Revenue.total_revenue_date_range(params['start'], params['end'])
```

Comment:
Might want to reconsider putting start and end as private param

However, I don't believe I am trying to protect a date someone is entering on their own. If the date doesn't exist, data would be null.